src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', (e) => {
+    resolve('Success');
+    document.body.appendChild(resolved);
```

Comment:
The variable `resolved` is used here before it is defined. Consider moving the creation and configuration of the `resolved` element before this line.